_book/04-advanced-topics/04-controlling-the-svg-output.md

```diff
@@ -1,11 +1,128 @@
+Up to Verovio 3.13 only integer MEI units are supported.
+{% endaside %}
+
+The example below shows the same file as above with the default A4 page size but with a unit value of `6.0`. More music will be rendered on a page because the staff size is smaller.
+
+{% include music-notation-only example="02-units" %}
+
+Changing the page dimension will increase the amount of music that fits on the page. The example below has been rendered with the default unit of `9.0` but a page height of `3050` and a page width of `2290`, a more typical paper size for sheet music than A4.
+
+{% include music-notation-only example="03-page-size" %}
+
+### Scaling
+
+The SVG output in Verovio can be scaled by using the `--scale` option. The option value is an integer representing a scaling percentage. It is `100` percent by default.
+
+When changing the scale option, Verovio will by default change the size of the output SVG image. For example, with the default page size and a scale option set to `50` percent, the resulting SVG image will have a size of **1485px** by **1050px**. The same amount of music will be engraved on the page as with the default scale value.
+
+In responsive environments, Verovio can be used to create user interfaces where the user can zoom in and out. This can be achieved by changing the scale and the page dimensions. Zooming out means increasing the page dimensions and reducing the scale by the same factor, and zooming in the opposite. For example, if the window in which the output of Verovio will be displayed is 1800px by 800px, these can be set as a page height and page width and Verovio will produce an SVG image that fits the window with the default scale value of 100 percent. To implement a zooming out function, for example by a factor 2, the page dimensions have to be changed to 3600 by 1600 and the scale to 50. The output SVG image will then still have a size that fits the window.
+
+Verovio has a `--scale-to-page-size` option that simplifies this process and that is recommended to be turned on in responsive environments. The advantage is that does not require the page dimensions to be calculated by the user when implementing zoom in or zoom out. With this option turned on, implementing a zoom function only requires the scale percentage to be changed and not the page dimensions. Keep in mind that the layout needs to be recalculated when the scale value is changed – see below.
+
+The example below shows a file rendered with a page height of `800`, a width of `1600` and the default scale of `100` percent.
+
+{% include music-notation-only example="04-responsive-page" %}
+
+The example below is the same file rendered with the same page dimensions as above, but with the option `--scale-to-page-size` turned on and a scale of `30` percent. The SVG image size remains the same but the amount of music rendered has increased accordingly.
+
+{% include music-notation-only example="05-responsive-page" %}
 
-### HTML5
+#### Layout recalculation
```

Comment:
These two paragraphs should probably be moved up above the examples, and not in a separate section, since they refer to the `--scale-to-page-size` option.